src/stepfunctions/steps/utils.py

```diff
@@ -45,3 +46,24 @@ def get_aws_partition():
             return cur_partition
 
     return cur_partition
+
+
+def merge_dicts(first, second, first_name, second_name):
+    """
+    Merges first and second dictionaries into the first one.
```

Comment:
Please document the args for this function in the docstring

Reply:
Will include args doc in the next commit